src/loader/LoaderAPI.ts

```diff
@@ -293,9 +293,13 @@ function $getRef(name: string, o: RefOpts): IReference {
         wildcard = validPath;
     }
     if (!file && !wildcard) {
-        // try folder index.js
+        // try index.js
         validPath = $pathJoin(filePath, "/", "index.js");
         file = pkg.f[validPath];
+
+        if (!file && filePath === ".") {
```

Comment:
Do we still need to modify the loader? Which case does it cover? `require(".")`?
Shouldn't it require index if exists?

Reply:
Yeah, So here is an example of it being hit
https://github.com/calebboyd/fuse-test

Adding a log in that if block (L301)
`console.log('loading: "', name, '" filePath: "', filePath, '" entry: "', pkg.s.entry, '"')`

-> 
```
loading: " ../ " filePath: " . " entry: " test.js "
loading: " ../../ " filePath: " . " entry: " test.js "
```

Reply:
It covers any fully relative path that resolves to the package.json in the pkg context